src/teaching-element/JoditHtml.vue

```diff
@@ -15,3 +15,58 @@ export default {
   }
 };
 </script>
+
+<style lang="scss">
+$borderSize: 6px;
+$tooltipColor: #455a64;
+
+.jodit_wysiwyg {
```

Comment:
A class needs to be added to the div with the `jodit_container` classes because of the fact that no styling in this library is scoped by Vue. So it affects all the Jodit instances (ones not used by this element) in the application that uses it. Which means that we need to do the scoping through CSS by having a wrapper class that only this block of HTML has. So give it a name like `te-jodit-html-content` and put all this CSS inside of that class.